app/window.go

```diff
@@ -895,6 +896,8 @@ func (w *Window) processEvent(d driver, e event.Event) bool {
 		w.decorations.Config = e2.Config
 		e2.Config = w.effectiveConfig()
 		w.out <- e2
+	case transfer.URLEvent:
+		w.out <- e2
```

Comment:
This is simple, but global. I believe `transfer.URLEvent`s should be routed like `transfer.DataEvent`. No?

Reply:
I'm not sure. In the case of `DataEvent` it can target a specific area of the UI, right? Like, drag/drop at specific zone. In the case of `URLEvent` that is somewhat external. At least in my case, that will require to have `SomeOp{Tag: tag}` in the `main`. However, that already happens with the "Back Button" (which is now on `InputOp`).

I don't know if that should be routed, and if it should be a new `Op` or use `TargetOp`? The `MIME` field is strange in that case. Is `MIME` the scheme? What happens if want the listen "all schemes"?

Reply:
I think (most/all) events should be routed, for composability of components. Without routing, you can't just use some widget and forget about its internals. With routing, you don't need to know that the widget listens for a particular dnd or file open.

> I don't know if that should be routed, and if it should be a new Op or use TargetOp? The MIME field is strange in that case. Is MIME the scheme? What happens if want the listen "all schemes"?

This is a problem for dnd as well. I imagine the MIME type be deferred from the file extension, falling back to "application/octet-stream". I also imagine you being able to register a range of types with say "image/*" or similar.

See also https://lists.sr.ht/~eliasnaur/gio-patches/patches/42549.

Reply:
But in that case what "mime" is the "scheme"? I mean, how I listen to all schemes?`MIME: "schemes/*`? How I listen to a specific scheme? `MIME: "schemes/myCustomScheme`? 

Also, what happens when the app is launched using one URL/Scheme? Because, in that specific case, I think you don't have `TargetOp` setup yet. For each new `TargetOp` it sends the "startup-url"?

Reply:
> But in that case what "mime" is the "scheme"? I mean, how I listen to all schemes?`MIME: "schemes/*`? How I listen to a specific scheme? `MIME: "schemes/myCustomScheme`?
> 

Good points. A `SchemeOp` seems appropriate.

> Also, what happens when the app is launched using one URL/Scheme? Because, in that specific case, I think you don't have `TargetOp` setup yet. For each new `TargetOp` it sends the "startup-url"?

The incoming URL must be retined by the `app` package until one frame has been processed, to give the program a chance to receive the URL. After that, it can be forgotten.

Reply:
I add a new commit with SchemeOp. I create a another `router/transfer.go`, since the current `TargetOp`seems to be integrated with `clip-area`.